### PR TITLE
Some OIDs used in Russian X.509 certificates.

### PR DESCRIPTION
**Pull request by beldmit**
OBJ_OGRNIP denotes a specific legal status of the certificate owner.
OBJ_classSignTool* denotes a level of certification of the software
created the certificate.

http://www.garant.ru/products/ipo/prime/doc/70033464/ is the relevant
link (in Russian).

### CODE REVIEW

File: crypto/objects/objects.txt [hunk 1]
```diff
@@ -1391,8 +1391,16 @@ id-tc26-gost-28147-constants 1	: id-tc26-gost-28147-param-Z : GOST 28147-89 TC26
 member-body 643 3 131 1 1	: INN	: INN
 member-body 643 100 1		: OGRN	: OGRN
 member-body 643 100 3		: SNILS	: SNILS
+member-body 643 100 5		: OGRNIP	: OGRNIP
 member-body 643 100 111	: subjectSignTool	: Signing Tool of Subject
 member-body 643 100 112	: issuerSignTool	: Signing Tool of Issuer
+member-body 643 100 113  : classSignTool   : Class of Signing Tool
```
Comment (mattcaswell) [inline crypto/objects/objects.txt]: I was expecting to see the link here in a comment

Comment (beldmit) [reply crypto/objects/objects.txt]: Sorry, my fault. I thought you wanted it in the commit message.

**Comment by beldmit** [comment]
Ping?

**Comment by beldmit** [comment]
Travis is happy

**Review by paulidale — APPROVED** [review]

**Comment by beldmit** [comment]
Merged. Thanks!

**Pull request closed without merging**